tests/test_modeling_tf_pegasus.py

```diff
@@ -231,22 +231,8 @@ def test_model_common_attributes(self):
                 name = model.get_bias()
                 assert name is None
 
-    @slow
-    def test_saved_model_with_hidden_states_output(self):
-        # TODO(JPLU, PVP) - fix this with s2s tf-serving PR
-        pass
-
-    @slow
-    def test_saved_model_with_attentions_output(self):
-        # TODO(JPLU, PVP) - fix this with s2s tf-serving PR
-        pass
-
     def test_saved_model_creation(self):
-        # TODO(JPLU, PVP) - fix this with s2s tf-serving PR
-        pass
-
-    def test_saved_model_creation_extended(self):
-        # TODO(JPLU, PVP) - fix this with s2s tf-serving PR
+        # This test is too long (>30sec) and makes fail the CI
```

Comment:
Is this bound to change? In the meantime, maybe we can put it as a slow test instead of skipping it?

Reply:
It is kind of useless because `test_saved_model_creation` is included in `test_saved_model_creation_extended` so it will be like doing twice the same test.

Reply:
Ok!